bin/Makefile.am

```diff
@@ -114,26 +114,29 @@ endif
 
 sstmac_clang_LDFLAGS = $(CLANG_LDFLAGS) \
  -Wl,-rpath,@CLANG_INSTALL_DIR@/lib \
- -lclangFrontend \
- -lclangFrontendTool \
- -lclangTooling \
- -lclangBasic \
+ -lclangAnalysis \
+ -lclangAST \
  -lclangASTMatchers \
+ -lclangBasic \
+ -lclangDriver \
+ -lclangEdit \
  -lclangFormat \
  -lclangFrontend \
- -lclangDriver \
+ -lclangFrontendTool \
+ -lclangLex \
  -lclangParse \
- -lclangSerialization \
+ -lclangRewrite \
  -lclangSema \
- -lclangEdit \
- -lclangAnalysis \
+ -lclangSerialization \
+ -lclangTooling \
  -lclangToolingCore \
- -lclangAST \
- -lclangRewrite \
- -lclangLex \
- -lclangBasic \
  @CLANG_LIBTOOLING_LIBS@ \
  @CLANG_LIBTOOLING_SYSTEM_LIBS@
+
+if CLANG_NEED_LIBCPP
+sstmac_clang_LDFLAGS += -lclang-cpp
```

Comment:
I still don't understand this, I use libc++ on ubuntu 16.04 and I don't need to add this.